apps/src/StudioApp.js

```diff
@@ -2759,7 +2760,8 @@ StudioApp.prototype.setPageConstants = function (config, appSpecificConstants) {
     config.level.markdownInstructions,
     config.locale,
     !!config.noInstructionsWhenCollapsed,
-    !!config.showInstructionsInTopPane
+    !!config.showInstructionsInTopPane,
+    !!config.level.inputOutputTable
```

Comment:
Why are you coercing `inputOutputTable` to boolean here, when it's described as a `number[][]` [here?](https://github.com/code-dot-org/code-dot-org/pull/8903/files#diff-800ff2833ea826c7994d8aabaf918fabR14)